similarity/src/main/java/edu/illinois/cs/cogcomp/sim/Datastore.java

```diff
@@ -0,0 +1,160 @@
+package edu.illinois.cs.cogcomp.sim;
```

Comment:
The reason I have this file is that I want the file downloaded to the "src/main/resources/" folder. In your datastore package, the file will download into System.getProperty("user.home"), but I can't specify "user.home" locations in my configuration file.